mmv1/templates/terraform/resource.erb

```diff
@@ -122,7 +122,8 @@ func resource<%= resource_name -%>() *schema.Resource {
             <% unless field.default_value.nil? -%>
               Default:  <%= go_literal(field.default_value) -%>,
             <% end -%>
-            },
+              Description: `<%= field.description.strip.gsub("`", "'") -%>`,
+          },
```

Comment:
The spacing on this end brace looks different, I think that's the cause of most of the diffs. The original 12 spaces lines up with L119.